website/siteConfig.js

```diff
@@ -14,7 +14,7 @@ const users = [
     caption: 'User1',
```

Comment:
Let's get rid of the whole `users` array and see if it works, since we don't have other projects involved with ours yet!